tools/cli/defs.go

```diff
@@ -65,6 +65,8 @@ const (
 	showErrorStackEnv    = `CADENCE_CLI_SHOW_STACKS`
 
 	searchAttrInputSeparator = "|"
+
+	managedFailoverDomainData = "IsManagedByCadence"
```

Comment:
This const probably should be defined by the server (in the common package for example) not by the CLI. CLI can then import and use this constant.

Reply:
updated.